js/render/draw_line.js

```diff
@@ -140,15 +137,22 @@ module.exports = function drawLine(painter, layer, posMatrix, tile) {
     var element = tile.buffers.lineElement;
     element.bind(gl);
 
+    gl.disableVertexAttribArray(shader.a_color);
+
     for (var i = 0; i < elementGroups.groups.length; i++) {
+        vertex.bind(gl);
```

Comment:
I reckon this is only necessary if we are actually binding a buffer to the attribute.